src/main/java/org/seqdoop/hadoop_bam/VCFFormat.java

```diff
@@ -59,4 +61,9 @@ public static VCFFormat inferFromData(final InputStream in) throws IOException {
 		}
 		return null;
 	}
+
+	public static boolean isBGZF(final InputStream in) throws IOException {
```

Comment:
Agreed that this is not really needed. I've removed the method and call isValidFile directly.